README.md

```diff
@@ -602,11 +602,11 @@ When the `/root/.mylogin.cnf` exists the environment variable `MYSQL_TEST_LOGIN_
 This is required if `create_root_user` and `create_root_login_file` are true. If `root_password` is 'UNSET', then `create_root_user` and `create_root_login_file` are assumed to be false --- that is, the MySQL root user and `/root/.mylogin.cnf` are not created.
 
 ```puppet
-class { '::mysql::server':
+class { 'mysql::server':
 root_password          => 'password',
 create_root_my_cnf     => false,
 create_root_login_file => true,
-login_file             => "puppet:///modules/${module_name}/mylogin.cnf",
+login_file             => 'puppet:///modules/mysql/mylogin.cnf',
```

Comment:
I don't think this change is correct.  I'm not a fan of `$module_name`, but here I think it refers to the `profile` module declaring `mysql::server` not to the `mysql` module itself.

Reply:
Good eye @alexjfisher

I reverted this change.